internal/api/saml/storage.go

```diff
@@ -380,9 +383,96 @@ func (p *Storage) getCustomAttributes(ctx context.Context, user *query.User, use
 			return nil, err
 		}
 	}
+
+	function := exec_repo.ID(domain.ExecutionTypeFunction, domain.ActionFunctionPreSAMLResponse.LocalizationKey())
```

Comment:
is it really the idea to execute the old and the new actions?

Reply:
If we want to check for the feature flag, that would mean an additional query, which happens like this as well to query the execution targets. 
My idea was to have both options running for the migration.

Reply:
Reasonable, let me look for the right place to document it. If we want to change this behaviour we would also need to change the already existing handling for request and response handling in the actions v2. The same for the handling of events in the future PR.